haskell-src/Concordium/Types/KonsensusV1.hs

```diff
@@ -0,0 +1,134 @@
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+-- |Types that are relevant only for endpoints exposed in 'ConsensusV1'.
+module Concordium.Types.KonsensusV1 where
```

Comment:
I think it might be best to rename to `Concordium.Types.Queries.KonsensusV1` to emphasise that these types exist for supporting queries, and are distinct to the types used internally by the consensus.